Resources/Audio/_RMC14/Weapons/Guns/Gunshots/attributions.yml

```
@@ -157,3 +157,8 @@
   license: "CC-BY-SA-3.0"
   copyright: "Taken from cmss13"
   source: "https://github.com/cmss13-devs/cmss13/blob/32730c731936c6737668ead7bde83a6cf296010f/sound/weapons/gun_mar40.ogg"
+
+- files: ["gun_ml66a_2.ogg"]
+  license: "CC0"
```

So that its recognized by the attribution validator script
```suggestion
  license: "CC0-1.0"
```